### PR TITLE
Updated filter-effects IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-06 21:25:40
Finished: 2018-03-06 21:37:06


<h2>Failing Jobs</h2>
<ul>

<li>chrome:dev</li>

<li>safari:11.0</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25391)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/350024007)

File: interfaces/filter-effects.idl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+interface SVGFETurbulenceElement : SVGElement {
```
Comment (foolip) [inline interfaces/filter-effects.idl]: https://drafts.fxtf.org/filter-effects/ has a cartload of IDL, so something gone awry here.

**Review by foolip — CHANGES_REQUESTED** [review]

File: interfaces/filter-effects.idl (hunk 1 above)
Comment (lukebjerring) [reply interfaces/filter-effects.idl]: Yep, this was a bug in the FOAM model which I switched too. PTAL.

**Review by foolip — CHANGES_REQUESTED** [review]
This'll require a test in css/filter-effects/ using the file, and it might be a bit of work. Could skip creating instances to reduce the burden a bit, or find a tag name -> interface name mapping somewhere to stamp out instances more mechanically.

**Comment by lukebjerring** [comment]
Test added. PTAL.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Oh, there's a failing MISSING-LINK lint. @gsnedders, how to deal with that for .any.js tests?

**Pull request merged**